wgengine/pendopen.go

```diff
@@ -86,6 +86,35 @@ func (e *userspaceEngine) trackOpenPreFilterIn(pp *packet.Parsed, t *tstun.Wrapp
 	return
 }
 
+var (
+	appleIPRange     = netip.MustParsePrefix("17.0.0.0/8")
+	canonicalIPRange = netip.MustParsePrefix("185.125.190.0/24")
```

Comment:
Should we be adding the other prefixes announced by [`AS41231`](https://bgp.he.net/AS41231)?

Reply:
Done. Added all. And a test.